ConceptMap/ConceptMap-Genomics-patient-foetusobservedsex.json

```diff
@@ -0,0 +1,75 @@
+{
+  "resourceType": "ConceptMap",
+  "id": "genomics-foetusobservedsex",
+  "url": "https://fhir.nhs.uk/ConceptMap/genomics-foetusobservedsex",
+  "version": "0.1.0",
+  "name": "GenomicsFoetusObservedSex",
+  "title": "Genomics Foetus Observed Sex",
+  "status": "draft",
+  "date": "2024-08-19T16:00:00.000Z",
+  "publisher": "NHS England",
+  "contact": [
+    {
+      "name": "NHS England",
+      "telecom": [
+        {
+          "system": "email",
+          "value": "interoperabilityteam@nhs.net"
+        }
+      ]
+    }
+  ],
+  "description": "A ConceptMap used for the translation and recording of concepts from MDS source data(code-options) to equivalent Snomed CT codes, and FHIR spec.",
+  "purpose": "ConceptMap used for the translation and recording of concepts for the patient gender if the patient is identify as fetal. This is intended for the element Patient.gender",
+  "copyright": "Copyright © 2023+ NHS England Licensed under the Apache License, Version 2.0 (the \\\"License\\\"); you may not use this file except in compliance with the License. You may obtain a copy of the License at  http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an \\\"AS IS\\\" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License. HL7® FHIR® standard Copyright © 2011+ HL7 The HL7® FHIR® standard is used under the FHIR license. You may obtain a copy of the FHIR license at  https://www.hl7.org/fhir/license.html.",
+  "targetUri": "http://hl7.org/fhir/ValueSet/administrative-gender",
+  "group": [
+    {
+      "target": "http://terminology.hl7.org/CodeSystem/v3-AdministrativeGender",
+      "element": [
+        {
+          "code": "female",
+          "display": "Female",
+          "target": [
+            {
+              "code": "F",
+              "display": "Female",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "male",
+          "display": "Male",
+          "target": [
+            {
+              "code": "M",
+              "display": "Male",
+              "equivalence": "equivalent"
+            }
+          ]
+        },
+        {
+          "code": "undetermined",
+          "display": "Undetermined",
+          "target": [
+            {
+              "equivalence": "unmatched"
```

Comment:
This should be mapped to other, with notes to state that this is an inexact match but is the only relevant code from the required codesystem